Source/WebInspectorUI/Versions/Inspector-iOS-14.0.json

```diff
@@ -3639,7 +3639,7 @@
         {
             "id": "Appearance",
             "type": "string",
-            "condition": "(defined(ENABLE_DARK_MODE_CSS) && ENABLE_DARK_MODE_CSS) || (defined(HAVE_OS_DARK_MODE_SUPPORT) && HAVE_OS_DARK_MODE_SUPPORT)",
+            "condition": "(defined(ENABLE_DARK_MODE_CSS) && ENABLE_DARK_MODE_CSS)",
```

Comment:
everything inside `Source/WebInspectorUI/Versions` exists for backwards compatibility and being able to look up the state of things in the past. so please do not edit them

Reply:
ooh, good to know! might be good to have a style checker rule for that at some point